tekton/src/pipelines/taskdefs/fvt-apps/manage.yml.j2

```diff
@@ -239,3 +239,14 @@
   # Update runAfter list with new PHASE 4 tests
   runAfter:
     - fvt-manage-startcenter-config
```

Comment:
@unnati-solanki-git I have noticed Direct Print is being tested in paralel with Work flow designer (which was supposed to run isolated, like start center) - and I have noriced, btw, my comment in line 211 is wrong, indicating start center is the last test... the last one should be workflow designer (all the Indudtry solution tests are configured to run after workflow).

We need to add direct print and smtp in a different place... do they have any restriction or can run in parallell with any test? Because if so, we can add them in different test phases (I will improve that by separating the different phases in different files later anyway).

Also, be sure to update comments on top of the different test phases, with the new tests we are adding.

Reply:
Hi @alequint direct-print and smtp both have dependency on setup only. I agree we should add them in different test phases.

Reply:
@unnati-solanki-git I did reorganize the pipeline and now let smtp as the latest task instead of workflow designer and move other tests to different phases (as I've noticed not all the tests depend on setup). It is already in this PR, but here is the commit FYI: https://github.com/ibm-mas/cli/commit/0ba73a547c3a50d5983fa0236eaa9576d7f68a42